draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2773,9 +2786,9 @@ uCDN-A                         dCDN-B                         dCDN-C
       </section>
    </section>
 
-   <section title="Other CI/T Objects" anchor="other-cit-objects">
+   <section title="Other CI/T Objects and Properties" anchor="other-cit-objects-and-properties">
    <t>
-	   This section describes common CI/T objects and their encodings.
+	   This section describes common CI/T objects and their encodings, that are used by multiple CI/T objects.
```

Comment:
As worded, "are used by multiple CI/T objects" is a non-restrictive clause, it would be more correct to use "which" instead of "that".

However, the sentence might be easier to read if we say something like, "This section describes common CI/T objects, which are used as part of the specification of other CI/T objects, and their encodings."

Reply:
Accepted as follows:
This section describes common CI/T objects, which are used as part of the specification of **several** other CI/T objects, and their encodings.

My idea was to place here objects that are used in multiple places in the CI/T object hierarchy.